### ISSUE TITLE
Example form.jl broken

### ISSUE THREAD

**Issue by ranjanan** [shashi/Escher.jl#120]
Here's the error trace:
```
UndefVarError: pad not defined
 in anonymous at /Users/ranjan/.julia/v0.4/Escher/src/cli/serve.jl:170
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/Mux.jl:15
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/Mux.jl:8
 in splitquery at /Users/ranjan/.julia/v0.4/Mux/src/basics.jl:28
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/Mux.jl:8
 in wcatch at /Users/ranjan/.julia/v0.4/Mux/src/websockets_integration.jl:12
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/Mux.jl:8
 in todict at /Users/ranjan/.julia/v0.4/Mux/src/basics.jl:21
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/Mux.jl:12 (repeats 2 times)
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/Mux.jl:8
 in anonymous at /Users/ranjan/.julia/v0.4/Mux/src/server.jl:38
 in handle at /Users/ranjan/.julia/v0.4/WebSockets/src/WebSockets.jl:382
 in on_message_complete at /Users/ranjan/.julia/v0.4/HttpServer/src/HttpServer.jl:393
 in on_message_complete at /Users/ranjan/.julia/v0.4/HttpServer/src/RequestParser.jl:104
 in http_parser_execute at /Users/ranjan/.julia/v0.4/HttpParser/src/HttpParser.jl:92
 in process_client at /Users/ranjan/.julia/v0.4/HttpServer/src/HttpServer.jl:365
 in anonymous at task.jl:447
```

**Comment by shashi** [comment]
This is what happens here:

1. You open a UI which uses Compose/Gadfly which also export `pad`
2. Julia decides that uses of `pad` must be qualified
3. Julia says that this variable is not defined

this example can be fixed by using Escher.pad

**Comment by ranjanan** [comment]
Oh okay, cool. I'll send a PR right now.

**Comment by randyzwitch** [comment]
Related #41

**Comment by ranjanan** [comment]
Since this is fixed by #121 I'm closing it.

**Comment by randyzwitch** [comment]
Thanks!
